src/main/java/com/catcher/core/service/ScheduleService.java

```diff
@@ -170,11 +170,12 @@ public void updateScheduleDetail(User user, Long scheduleDetailId, UpdateSchedul
     }
 
     @Transactional
-    public void deleteScheduleDetail(User user, Long scheduleDetailId){
+    public void deleteScheduleDetail(User user, Long scheduleDetailId) {
         getScheduleDetail(user, scheduleDetailId);
         scheduleDetailRepository.deleteScheduleDetail(user, scheduleDetailId);
     }
 
+    @Transactional
```

Comment:
이 부분은 굳이 트랜잭션 안붙여도 될것같긴한데 붙인다면 readonly를 적용하는 게 좋을거같아용
```suggestion
    @Transactional(readonly = true)
```